crx2rnx/Cargo.toml

```diff
@@ -12,4 +12,4 @@ readme = "README.md"
 
 [dependencies]
 clap = { version = "4", features = ["derive", "color"] }
-rinex = { version = "0.13.1", features = ["serde"] }
+rinex = { path = "../rinex", version = "=0.14.0", features = ["serde"] }
```

Comment:
so if I understand correctly, workspace definitions can be inherited by sub Cargo.toml files

Reply:
They're just a way to write `rinex = { path = "rinex", version = "0.14.0" }` once instead of once for every crate depending on it.

Reply:
> Edit: nope. workspace.dependencies can only distribute packages defined outside of this workspace.

But it works [here](https://github.com/rust-lang/rust-analyzer/blob/master/Cargo.toml#L47)?

Reply:
I don't see how this can apply here.

The root cargo file can distribute one implementation of one local crate.
We don't have candidates for that: we don't have one crate used by all the others.
It could partially apply to rinex, but inner crates are highly dependent on the crate features implemented. 
I'm not sure what the cargo workspace will distribute, and you don't want for instance, to build "rinex --full" if you don't need it